daemon/rpc_test.go

```diff
@@ -14,6 +14,8 @@ import (
 	"testing"
 	"time"
 
+	testsession "github.com/NordSecurity/nordvpn-linux/test/mock/session"
```

Comment:
[small] in other places this is called `mocksession`, would be nice to be consistent maybe, but not a big deal